vendor/bat-native-ledger/src/bat/ledger/internal/properties/reconcile_payload_properties.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_RECONCILE_PAYLOAD_PROPERTIES_H_
+#define BRAVELEDGER_RECONCILE_PAYLOAD_PROPERTIES_H_
+
+#include <string>
+
+#include "bat/ledger/internal/properties/unsigned_tx_properties.h"
+
+namespace ledger {
+
+struct ReconcilePayloadProperties {
+  ReconcilePayloadProperties();
+  ReconcilePayloadProperties(
+      const ReconcilePayloadProperties& properties);
+  ~ReconcilePayloadProperties();
+
+  bool operator==(
+      const ReconcilePayloadProperties& rhs) const;
+
+  bool operator!=(
+      const ReconcilePayloadProperties& rhs) const;
+
+  std::string request_type;
```

Comment:
based on the fact that all properties in this struct has request, should we rename struct to have Request in it and drop it from variables. For example `ReconcileRequestProperties`

Reply:
Fixed